### PR TITLE
Add bash script for setting up Ansible control node

### PR DESCRIPTION
**Pull request by nasirkamal**
Code for setting up ansible target and control nodes. It automates the process of user creation, key sharing and packages installation.

### CODE REVIEW

File: setup-ansible-node/vars.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cnode_username=ansible1
```
Comment (usamaahmadkhan) [inline setup-ansible-node/vars.txt:1]: defaults should be ansible not ansible1 as described in the README

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

File: setup-ansible-node/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	- Generate ssh keys (as user cnode_username)
+	- Copy ssh public key to the target node <tnode_username>@<tnode_hostname>
+	- Install Packages (Python3, Ansible)
+
+
+For creating a single node (control + target), vars.txt needs to be populated with the proper values. An example is given below:
+
+vars.txt
+--------
+
```
Comment (usamaahmadkhan) [inline setup-ansible-node/README.md:37]: Plz use code block syntax for the below values. It doesn't render properly

File: setup-ansible-node/vars.txt (hunk 1 above)
Comment (nasirkamal) [reply setup-ansible-node/vars.txt]: Done

File: setup-ansible-node/README.md (hunk 2 above)
Comment (nasirkamal) [reply setup-ansible-node/README.md]: Done

**Pull request merged**